src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carCostPerDay = 40;
+  const mediumRentTermDiscount = 20;
+  const longRentTermDiscount = 50;
+  const price = carCostPerDay * days;
+
+  if (days < 3) {
+    return price;
+  }
+
+  if (days >= 3 && days <= 6) {
+    return price - mediumRentTermDiscount;
+  }
+
+  if (days >= 7) {
```

Comment:
The condition 'days >= 7' is redundant because of the previous conditions. If the code execution reaches this point, it means that 'days' is already 7 or more. You can remove the condition check.